### ISSUE TITLE
Gantt throws on invalid constraint

### ISSUE THREAD

**Issue by bmblb** [bryntum/support#1770]
Steps to reproduce on advanced demo:
1. add this config to task 14:
```
"constraintType": "finishnolaterthan",
"constraintDate": "2019-01-14"
```
Expected: constraint is removed, warning logged saying constrant is invalid, demo works

Got: exception, demo doesn't work
```
Uncaught (in promise) TypeError: Cannot read property 'get' of null
    at EngineRevision.getLatestEntryFor (Revision.js:18)
    at EngineTransaction.readPrevious (Transaction.js:236)
    at EngineTransaction.readCurrentOrProposedOrPrevious (Transaction.js:222)
    at TemplateClass.getFromGraph (ModelFieldAtom.js:58)
    at ProjectModel.get (Entity.js:151)
    at GanttToolbar.updateStartDateField (GanttToolbar.js:339)
    at ProjectModel.trigger (Events.js:1057)
    at EngineReplica.finalizeCommitAsync (Replica.js:89)
    at EngineReplica.doCommitAsync (Graph.js:219)
    at async EngineReplica.doCommitAsync (Graph.js:221)
```

**Comment by canonic-epicure** [comment]
This one is basically a duplicate of the "Need to have conflicts processing"

**Comment by canonic-epicure** [comment]
Not valid since 5.0 - conflicts processing has been implemented in 5.0.
